daemoncfg/daemon_config_test.go

```diff
@@ -30,49 +29,49 @@ func TestGetDaemonEndpoints1(t *testing.T) { // default address set to udp and t
 	assert.Equal(t, dEndpt.TCPAddr, tcpEndpt)
 }
 
-func TestGetDaemonEndpoints2(t *testing.T) { // default address set to udp and tcp
-	udpAddr := "127.0.0.1:4000"
-	tcpAddr := "127.0.0.1:5000"
-	udpEndpt, _ := resolveUDPAddr(udpAddr)
-	tcpEndpt, _ := resolveTCPAddr(tcpAddr)
+// func TestGetDaemonEndpoints2(t *testing.T) { // default address set to udp and tcp
```

Comment:
Any reason to comment out functions TestGetDaemonEndpoints2, TestGetDaemonEndpointsFromEnv1, TestGetDaemonEndpointsFromEnv2 ?

Reply:
added by https://github.com/aws/aws-xray-sdk-go/pull/177/commits/645b4bda883f350cabc08504f4a751e7e4c621e2